### PR TITLE
Fixed ReturnBookControl & ReturnBookUI

### PR DESCRIPTION
**Pull request by declandaman**
Review style/formatting changes

### CODE REVIEW

File: src/ReturnBookUI.java [hunk 1]
```diff
@@ -1,87 +1,81 @@
...
-	}
+                default: {
+                    output("Unhandled state");
+                    throw new RuntimeException("ReturnBookUI : unhandled state :" + state);
+                }
+            }
+        }
+    }
 
+    private String input(String prompt) {
```
Comment (Black-Clone) [inline src/ReturnBookUI.java]: D13 - Change input() -> getInput() (differentiate from the input variable used in the same class file)

File: src/ReturnBookControl.java [hunk 2]
```diff
@@ -1,77 +1,72 @@
...
+        currentLoan = library.getLoanByBookId(bookId);
+        double overDueFine = 0.0;
+        if (currentLoan.isOverDue()) {
+            overDueFine = library.calculateOverDueFine(currentLoan);
+        }
+        ui.display("Inspecting");
+        ui.display(currentBook.toString());
+        ui.display(currentLoan.toString());
+        if (currentLoan.isOverDue()) {
+            ui.display(String.format("\nOverdue fine : $%.2f", overDueFine));
```
Comment (Black-Clone) [inline src/ReturnBookControl.java]: D4 - Change this line to remove method call within argument list of method call e.g.             
48 String overDueFineMessage = String.format("\nOverdue fine : $%.2f", overDueFine);
49 ui.display(overDueFineMessage);

File: src/ReturnBookUI.java [hunk 3]
```diff
@@ -1,87 +1,81 @@
...
+                    String bookStr = input("Scan Book (<enter> completes): ");
+                    if (bookStr.length() == 0) {
+                        control.scanningComplete();
+                    } 
+                    else {
+                        try {
+                            int bookId = Integer.valueOf(bookStr).intValue();
+                            control.bookScanned(bookId);
+                        } 
+                        catch (NumberFormatException e) {
```
Comment (Black-Clone) [inline src/ReturnBookUI.java]: D9 - Change e -> error

File: src/ReturnBookControl.java [hunk 4]
```diff
@@ -1,33 +1,42 @@
...
-        if (!state.equals(CONTROL_STATE.READY)) {
+        if (!state.equals(ControlState.READY)) {
             throw new RuntimeException("ReturnBookControl: cannot call bookScanned except in READY state");
         }
+        
         Book currentBook = library.book(bookId);
+        
         if (currentBook == null) {
             ui.display("Invalid Book Id");
             return;
```
Comment (Black-Clone) [inline src/ReturnBookControl.java]: Add a new line in between lines 43 and 44 to make spacing uniform throughout file.

File: src/ReturnBookUi.java [hunk 5]
```diff
@@ -1,22 +1,47 @@
...
+
+    
+    public void setState(UiState state) {
+        this.state = state;
+    }
+    
+    
     public void run() {
         output("Return Book Use Case UI\n");
         while (true) {
```
Comment (Black-Clone) [inline src/ReturnBookUi.java]: D15 - Add a new line in between lines 48 and 49 to make spacing uniform throughout file.

File: src/ReturnBookUi.java [hunk 6]
```diff
@@ -25,25 +50,26 @@ public void run() {
                     break;
                 }
```
Comment (Black-Clone) [inline src/ReturnBookUi.java]: Add a new line in between lines 51 and 52 to make spacing uniform throughout file.

File: src/ReturnBookUi.java [hunk 7]
```diff
@@ -25,25 +50,26 @@ public void run() {
...
                 case READY: {
-                    String bookStr = input("Scan Book (<enter> completes): ");
-                    if (bookStr.length() == 0) {
+                    String bookIdString = getInput("Scan Book (<enter> completes): ");
```
Comment (Black-Clone) [inline src/ReturnBookUi.java]: Add a new line in between lines 53 and 54 to make spacing uniform throughout file.

File: src/ReturnBookUi.java [hunk 8]
```diff
@@ -25,25 +50,26 @@ public void run() {
...
+                            output("Invalid Book Id");
                         }
                     }
                     break;
                 }
+                
                 case INSPECTING: {
-                    String ans = input("Is book damaged? (Y/N): ");
+                    String answer = getInput("Is book damaged? (Y/N): ");
                     boolean isDamaged = false;
```
Comment (Black-Clone) [inline src/ReturnBookUi.java]: Add a new line in between lines 71 and 72 to make spacing uniform throughout file.

File: src/ReturnBookControl.java [hunk 9]
```diff
@@ -1,33 +1,42 @@
...
+        state = ControlState.INITIALISED;
     }
 
     
-    public void setUI(ReturnBookUI ui) {
-        if (!state.equals(CONTROL_STATE.INITIALISED)) {
+    public void setUi(ReturnBookUi ui) {
+        if (!state.equals(ControlState.INITIALISED)) {
             throw new RuntimeException("ReturnBookControl: cannot call setUI except in INITIALISED state");
         }
```
Comment (Black-Clone) [inline src/ReturnBookControl.java]: Add a new line in between lines 24 and 25 to make spacing uniform throughout file.

File: src/ReturnBookUi.java [hunk 10]
```diff
@@ -25,25 +50,26 @@ public void run() {
...
-                    if (ans.toUpperCase().equals("Y")) {
+                    if (answer.toUpperCase().equals("Y")) {
                         isDamaged = true;
                     }
```
Comment (Black-Clone) [inline src/ReturnBookUi.java]: Add a new line in between lines 74 and75 to make spacing uniform throughout file.

**Comment by declandaman** [comment]
Successfully merged the fixed ReturnBookControl and ReturnBookUi files, as well as a Main copy with the updated class name.

**Pull request merged**